### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by aagea**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/aagea/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+Over on the right is a panel for line notes that you can connect to a
+line of code in the diff. You can also switch to the conversation
+to see comments on the whole pull request.
+
+Line notes have flags to indicate what is going on in the code,
+too. Flags are as follows:
+
+`['!fix', '!resolve']`
+- Something in the code needs to be reviewed and fixed before
+  the pull request is merged.
```
Comment (aagea) [inline ReadMe.md:17]: Pure shit! !fix

Comment (aagea) [inline ReadMe.md:17]: Okay !resolved

**Comment by aagea** [comment]
hola

**Pull request merged**